examples/single_project/README.md

```diff
@@ -0,0 +1,61 @@
+This repo.yaml file contains a workflow specification to use Infracost with a single project. It uses `infracost comment` to post Infracost results to your PR. 
+
+## Running with GitHub
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) dockerhub image
+2. You'll need to pass the following custom env vars into the container
+   ```sh
+   GITHUB_TOKEN=<your-github-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+       pre_workflow_hooks:
+         # Clean up any files left over from the last run
+         - run: rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json
+   workflows:
+     terraform-infracost:
+       plan:
+         steps:
+           - init
+           - plan
+           # Run Infracost breakdown and save to a tempfile, namespaced by this project, PR, workspace and dir
+           - run: infracost breakdown --path=$PLANFILE --format=json --log-level=info --out-file=/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json
+           # Use Infracost comment to create a comment containing the results for this project.
+           - run: infracost comment github --repo $BASE_REPO_OWNER/$BASE_REPO_NAME --pull-request $PULL_NUM --path /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json --github-token $GITHUB_TOKEN
+   ```
+4. Restart the atlantis application with the new env vars and config 
+5. Send a pull request in GitHub to change something in TF, the Infracost pull request comment should be added.
+
+## Running with Gitlab
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) dockerhub image
+2. You'll need to pass the following custom env vars into the container
+   ```sh
+   GITLAB_TOKEN=<your-gitlab-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+       pre_workflow_hooks:
+         # Clean up any files left over from the last run
+         - run: rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json
```

Comment:
Should we do this at the end of the workflow now instead of here? I think we only needed it here for the multi-project example before since we didn't have a post workflow hook.

Reply:
Yeah let's move to the end of the run